source/MaterialXGenShader/GenOptions.h

```diff
@@ -44,12 +42,6 @@ class GenOptions
     /// code fragments will be generated for the shader and
     /// the surface will be fully opaque.
     bool hwTransparency;
-
-    /// An optional override for the target color space
-    std::string targetColorSpaceOverride;
-
-    /// Sets whether to perform a validation check before generation
```

Comment:
Hey Bernerd, I think I missed the code review on this because I didn't notice it until now. For validation I think it's better to keep that separate from shader generation. If an element is not valid you should not try to perform shader generation on it. And since validating an element is simple, just call element->validate(), I see no good reason for adding this in as a feature and option inside shader generation. So I would prefer to keep it outside.

For most applications validation should happen directly at graph construction time, to let users know directly they are creating an invalid construct (ideally the editor should prevent you from doing that). So for most cases this extra logic and option setting will not be used and just adds complexity.

Reply:
Okay. I thought when we talked about this we wanted it inside as an option. My misinterpretation.
Makes sense to leave it outside.